src/modules/75-cd/components/PipelineSteps/Common/Terragrunt/EditView/TerragruntEditView.tsx

```diff
@@ -693,7 +694,7 @@ export default function TerragruntEitView(
                                   {backendConfigFilePath && (
                                     <>
                                       <Text font="normal" lineClamp={1} width={200}>
-                                        {backendConfigFilePath}
+                                        /{backendConfigFilePath}
```

Comment:
Is `/` intentional. If yes, use 
```suggestion
                                        {`/${backendConfigFilePath}`}
```